authorize/authorize.go

```diff
@@ -63,3 +65,10 @@ func NewIdentityWhitelist(policies []policy.Policy, admins []string) IdentityVal
 func (a *Authorize) ValidIdentity(route string, identity *Identity) bool {
 	return a.identityAccess.Valid(route, identity)
 }
+
+// UpdateOptions updates internal structres based on config.Options
+func (a *Authorize) UpdateOptions(o *config.Options) error {
```

Comment:
Should this be returning an error ever?

Reply:
Not in authorize as written, but I believe proxy can.  It kind of depends on what the services are doing when you tell them to update.  Since they're in charge of option validation and then doing something with it, it is possible that they will have an issue and want to signal it back up the call stack.  I'm open to changing the interface spec if you don't think it adds value, though.

Reply:
No, I think that makes sense.